pkg/zcache/local_cache.go

```diff
@@ -56,14 +57,18 @@ func (item CacheItem) IsExpired() bool {
 
 type LocalCache interface {
 	ZCache
+	startCleanupProcess()
 }
 
 type localCache struct {
-	client         *bigcache.BigCache
+	client         *ristretto.Cache
 	prefix         string
 	logger         *logger.Logger
 	metricsServer  metrics.TaskMetrics
 	cleanupProcess CleanupProcess
+	deleteHits     uint64
+	deleteMisses   uint64
+	keysList       []string
```

Comment:
I don't see keyList being filled up at any point in the code. I think there is an error here.